### PR TITLE
fix: UI layout

### PR DESCRIPTION
**Pull request by krzysztofzuraw**
I want to merge this change because it fixes a bunch of UI related stuff

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
APPS_MARKETPLACE_API_URI=https://apps.staging.saleor.io/api/v2/saleor-apps
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?

To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=1

### CODE REVIEW

File: src/giftCards/GiftCardSettings/GiftCardExpirySettingsCard/GiftCardExpirySettingsCard.tsx [hunk 1]
```diff
@@ -16,17 +17,30 @@ export interface GiftCardExpirySettingsCardProps
...
 }) => {
   const intl = useIntl();
+  const classes = useStyles({});
 
   return (
     <Card data-test-id="gift-card-settings">
-      <CardTitle title={intl.formatMessage(messages.expiryDateTitle)} />
+      <CardTitle
+        title={intl.formatMessage(messages.expiryDateTitle)}
+        className={classes.cardTitle}
```
Comment (poulch) [inline src/giftCards/GiftCardSettings/GiftCardExpirySettingsCard/GiftCardExpirySettingsCard.tsx:42]: question: why not use here sprinkles with 0 padding ?

File: src/giftCards/GiftCardUpdate/GiftCardHistory/styles.ts [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
 const useStyles = makeStyles(
   theme => ({
     header: {
       fontWeight: 500,
       marginBottom: theme.spacing(1),
     },
-    root: { marginTop: theme.spacing(4) },
+    root: {
+      marginTop: theme.spacing(4),
```
Comment (poulch) [inline src/giftCards/GiftCardUpdate/GiftCardHistory/styles.ts:11]: question: why this spacing is from theme?

File: src/home/components/HomeActivityCard/HomeActivityCard.tsx [hunk 3]
```diff
@@ -56,47 +54,48 @@ const HomeActivityCard: React.FC<HomeActivityCardProps> = props => {
...
-        className={classes.cardTitle}
       />
-      <List dense={true}>
-        {renderCollection(
-          activities,
-          (activity, activityId) => (
-            <ListItem key={activityId} className={classes.listItem}>
-              {activity ? (
+      <CardContent>
+        <List dense={true}>
```
Comment (poulch) [inline src/home/components/HomeActivityCard/HomeActivityCard.tsx:59]:
```suggestion
        <List dense>
```

**Review by poulch — APPROVED** [review]
LGTM

File: src/giftCards/GiftCardSettings/GiftCardExpirySettingsCard/GiftCardExpirySettingsCard.tsx (hunk 1 above)
Comment (krzysztofzuraw) [reply src/giftCards/GiftCardSettings/GiftCardExpirySettingsCard/GiftCardExpirySettingsCard.tsx:42]: Too low specificity:
CleanShot 2023-02-14 at 15 37 06@2x

**Review by Droniu — APPROVED** [review]

**Pull request merged**